layouts/partials/home/family/v1.html

```diff
@@ -0,0 +1,27 @@
+{{ $title    := site.Params.v1.title }}
+{{ $subtitle := site.Params.v1.subtitle | markdownify }}
+{{ $img      := printf "img/%s" site.Params.v1.image | relURL }}
+{{ $buttons  := site.Params.v1.buttons }}
+<section id="v1" class="container box p-6 my-6">
+  <div class="columns is-vcentered">
+    <div class="column is-half">
+      <div class="content has-text-left">
+        <h1 class="title is-2">{{ $title }}</h1>
+        <div class="subtitle is-5">
+          {{ $subtitle }}
+        </div>
+        <div class="buttons">
+          {{ range $buttons }}
+          <a href="{{ .url }}" class="button is-medium is-{{ .color | default "white" }}">{{ .text }}</a>
+          {{ end }}
+        </div>
+      </div>
+    </div>
+    <div class="column is-half is-hidden-mobile">
+      <figure class="image">
+        <img src="{{ $img }}" alt="{{ $title }} illustration">
+      </figure>
+    </div>
+  </div>
+</div>
+</section>
```

Comment:
I think the styling of this block should be the same as the one we have for Flagger (minus the background and other "green" styling elements), so that the space between the blocks is OK.

Reply:
AFAICT it is: https://gist.github.com/dholbach/fcde6bb18f43c0f04b1bbf2d0ffc24f3

Reply:
Looks like something else is off then, or the cache isn't cleared properly? See first screenshot (current), versus second screenshot (local in-browser modification).

![Wrong placement section](https://user-images.githubusercontent.com/10063039/109021516-3e3bcb00-76bb-11eb-987b-fa7ee3a1da0e.png)
![Right placement section](https://user-images.githubusercontent.com/10063039/109021529-409e2500-76bb-11eb-95a1-ad73c2d9958d.png)

Reply:
:heavy_check_mark: ca2209d

Reply:
Awesome, thank you Daniel :stars: